commands/outbox.go

```diff
@@ -0,0 +1,103 @@
+package commands
+
+import (
+	"io"
+
+	"gx/ipfs/Qmde5VP1qUkyQXKCfmEUA7bP64V2HAptbJ7phuPp7jXWwg/go-ipfs-cmdkit"
+	"gx/ipfs/Qmf46mr235gtyxizkKUkTH5fo62Thza2zwXR4DWC7rkoqF/go-ipfs-cmds"
+
+	"github.com/filecoin-project/go-filecoin/address"
+	"github.com/filecoin-project/go-filecoin/core"
+)
+
+var outboxCmd = &cmds.Command{
+	Helptext: cmdkit.HelpText{
+		Tagline: "View and manipulate the outbound message queue",
+	},
+	Subcommands: map[string]*cmds.Command{
+		"ls":    outboxLsCmd,
```

Comment:
i know its pedantic but lexicographic order as these things grow is always helpful